tools/interop_matrix/client_matrix.py

```diff
@@ -18,6 +18,7 @@
 
 def get_github_repo(lang):
     return {
+        'dart': 'git@github.com:dart-lang/grpc-dart.git',
```

Comment:
Is there a release to test? If not, it's better to add this when there is one.

Reply:
Define release? We're currently at v0.2.1 on Pub: https://pub.dartlang.org/packages/grpc.